src/obp_accounting_sdk/constants.py

```diff
@@ -2,6 +2,8 @@
 
 from enum import StrEnum, auto
 
+HEARTBEAT_INTERVAL = 30
```

Comment:
To be discussed, maybe it can be convenient to make HEARTBEAT_INTERVAL configurable with an env variable